kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/BaseSiteService.java

```diff
@@ -2939,6 +3017,47 @@ protected void disableAuthorizationGroup(String ref)
 		 */
 		public void saveInfo(String siteId, String description, String infoUrl);
 
+		/**
+		 * Unpublish the site by simply unsetting the PUBLISHED flag
+		 * @param siteId
+		 *        The ID of the site to unpublish
+		 * @param modifedBy
+		 *        User who is unpublishing the site (as a userID)
+		 * @param modifiedOn
+		 *        Time that the site is unpublished
+		 */
+		public void unpublish(String siteId, String modifiedBy, Time modifiedOn);
```

Comment:
why the need for this method given:
public void unpublish(List<String> siteIds, String modifiedBy, Time modifiedOn);